### PR TITLE
[Feature] Draggable PiP

### PR DESCRIPTION
**Pull request by palatter**
## Purpose
Allow the PiP to be draggable around the call screen container.

## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingView.swift [hunk 1]
```diff
@@ -49,50 +51,77 @@ struct CallingView: View {
...
+                        .accessibilitySortPriority(1)
+                        .accessibilityHidden(viewModel.isLobbyOverlayDisplayed)
+                }
+                .onAppear {
+                    if self.pipPosition == nil {
+                        self.pipPosition = getInitialPipPosition(containerBounds: geometry.frame(in: .local))
+                    }
+                }
+                .onChange(of: geometry.size) { _ in
+                    self.pipPosition = getInitialPipPosition(containerBounds: geometry.frame(in: .local))
```
Comment (jimchou-dev) [inline AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingView.swift:72]: When pip is dragged and rotated back to portrait the pip seems to reset to the location middle right of the screen, is this intentional or is it suppose to go back to bottom right of the screen?
image

Comment (palatter) [reply AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/CallingView.swift:81]: Looks like the onSizeChanged isn't triggered when rotating that specific way on an iphone. So I have added back in my on Rotation solution. With both of these, it should cover all cases.

**Review by jimchou-dev — APPROVED** [review]
Tested, LGTM

**Review by JoshuaLai — APPROVED** [review]
Tested with a bunch of different scenarios and different outage scenarios as well and it looks good. Checked the memory as well looks like no new leaks

**Pull request merged**